examples/conv_lstm.py

```diff
@@ -8,11 +8,18 @@
 from keras.layers.normalization import BatchNormalization
 import numpy as np
 import pylab as plt
+from keras import backend as K
 
 # We create a layer which take as input movies of shape
 # (n_frames, width, height, channels) and returns a movie
 # of identical shape.
 
+
+if K.backend() == 'cntk':
+    raise RuntimeError('CNTK could not run this example, due to the missing support for padding '
+                       'with non-specified input shape. To run it with CNTK, please update the '
+                       'input_shape in top COnvLSTM2D layer from (None, 40, 40, 1) to (15, 40, 40, 1)')
```

Comment:
Because this is a beta version of cntk backend, parts of the examples are not supported. Those "if" codes were here to give user better message. After consider the impact, we revert the changes in examples, instead we provide better error message in cntk_backend, to avoid the complexity for keras management.

CNTK will fix the padding issue in GA version of keras backend.